src/common/types.ts

```diff
@@ -0,0 +1,83 @@
+import { PublicKey } from "@solana/web3.js"
+import { BN } from "@project-serum/anchor"
+
+/** Token account state as stored by the program */
+export enum AccountState {
+  Uninitialized = 0,
+  Initialized = 1,
+  Frozen = 2
+}
+
+/** Custom RawAccount with BN. Token account as stored by the program */
+export interface RawTokenAccountInfo {
+  mint: PublicKey
+  owner: PublicKey
+  amount: BN
+  delegateOption: 1 | 0
+  delegate: PublicKey
+  state: AccountState
+  isNativeOption: 1 | 0
+  isNative: BN
+  delegatedAmount: BN
+  closeAuthorityOption: 1 | 0
+  closeAuthority: PublicKey
+}
+
+/** Custom Account Interface with BN. Information about a token account*/
+export interface TokenAccountInfo {
```

Comment:
You'll have to not step on spl-tokens toes unfortunately
```suggestion
export interface JetTokenAccountInfo {
```